### PR TITLE
Added website id to the notification message schema

### PR DESCRIPTION
**Pull request by jankoulaga**

### CODE REVIEW

**Comment by khalidjaz** [comment]
👍

File: schemas/com.liveintent/retargeting_notification/jsonschema/1-0-0 [hunk 1]
```diff
@@ -23,5 +26,5 @@
       "type": "string"
     }
   },
-  "required": ["app_id", "domain_id", "collector_timestamp"]
+  "required": ["app_id", "website_id", "domain_id", "collector_timestamp"]
```
Comment (kazjote) [inline schemas/com.liveintent/retargeting_notification/jsonschema/1-0-0]: Major version should be bumped as adding new required field makes it not backwards compatible.

Comment (jankoulaga) [inline schemas/com.liveintent/retargeting_notification/jsonschema/1-0-0]: What does that mean? Renaming the file or adding a new one?

**Comment by jankoulaga** [comment]
@kazjote Are you ok with it now?

**Comment by kazjote** [comment]
Looks good now :+1:

**Pull request merged**